apps/web/src/lib/elastic/client.ts

```diff
@@ -0,0 +1,9 @@
+import { Client } from '@elastic/elasticsearch'
+
+export default new Client({
+  node: process.env.ELASTIC_URL,
+  auth: {
+    username: process.env.ELASTIC_USERNAME!,
+    password: process.env.ELASTIC_PASSWORD!,
```

Comment:
Don't use `process` use `env`
https://github.com/latitude-dev/latitude-llm/blob/main/apps/web/drizzle.config.ts

Is a type safe version